src/bytes.ts

```diff
@@ -105,30 +105,33 @@ const stripZeros = function(a: any): Buffer | number[] | string {
 }
 
 /**
- * Attempts to turn a value into a `Buffer`. As input it supports `Buffer`, `String`, `Number`, null/undefined, `BN` and other objects with a `toArray()` method.
+ * Attempts to turn a value into a `Buffer`.
+ * Inputs supported: `Buffer`, `String`, `Number`, null/undefined, `BN` and other objects with a `toArray()` or `toBuffer()` method.
  * @param v the value
  */
 export const toBuffer = function(v: any): Buffer {
```

Comment:
For completeness, we should set the type of `v` from `any` to all the imported inputs.

Reply:
good idea, done in https://github.com/ethereumjs/ethereumjs-util/pull/277/commits/e7aa906eb8ef53f5ec0c2581bafed1d69cec60d5